source/content.css

```diff
@@ -860,12 +860,11 @@ body > .footer li a {
 	align-items: flex-start !important;
 }
 /* Align between event and repo description */
-.dashboard .py-3 .avatar {
+.dashboard .flex-items-baseline .avatar {
 	margin-top: 7px;
 }
 /* Hide duplicate repo name */
-.dashboard .flex-items-baseline .f3,
-.dashboard .flex-items-baseline .f4 {
+.dashboard .flex-items-baseline .text-bold.text-gray-dark {
```

Comment:
Some events still use `f3`, like `repo created`

Just add the selector, don't replace it